lib/vmdb/loggers.rb

```diff
@@ -32,15 +31,19 @@ def self.apply_config(config)
 
     def self.create_logger(log_file_name, logger_class = VMDBLogger)
       log_file = ManageIQ.root.join("log", log_file_name)
-      logger_class.new(log_file).tap do |logger|
-        logger.extend(ActiveSupport::Logger.broadcast($container_log)) if $container_log
-        logger.extend(ActiveSupport::Logger.broadcast($journald_log))  if $journald_log
+      progname = File.basename(log_file_name, ".*")
+
+      logger_class.new(log_file, progname: progname).tap do |logger|
+        broadcast_logger = create_broadcast_logger
+        if broadcast_logger
+          logger.extend(ActiveSupport::Logger.broadcast(broadcast_logger))
+          logger.instance_variable_set(:@broadcast_logger, broadcast_logger)
```

Comment:
do we know if this is limited to certain rails versions?

Reply:
This is a hack for now - I couldn't find a way to "extract" the internal logger because of the way ActiveSupport::Logger.broadcast works - I'm sure there's a way, I just can't figure it out.

Reply:
I was considering that - now that you mentioned it I'll do it (with a comment too)

Reply:
Done.